docs/index.bs

```diff
@@ -3456,6 +3461,57 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
           1. Return true.
   </section>
 
+  <section algorithm>
+    <h3 id="check-router-registration-limit"><dfn>Check Router Registration Limit</dfn></h3>
+
+      : Input
+      :: |routerRules|, a [=list of router rules=]
+      : Output
+      :: a boolean
+
+      Note: Router conditions can be complex and nested using {{RouterCondition/_or}} and {{RouterCondition/not}}. To prevent excessive processing, this algorithm introduces two limits. First, the total number of conditions, counting all nested conditions, cannot exceed 1024. Second, the nesting depth is limited to 10 levels to avoid exponential computation.
+
+      1. Let |result| be a [=count router condition result=].
+      1. Set |result|'s [=count router condition result/total count=] to 0.
```

Comment:
What is the intention to set "total count" to 0 here instead of 1, and omit increment one in count-router-inner-conditions?

Reply:
IIUC, this line is the initializing part for "total count". The actual condition node counting is done inside count-router-inner-conditions. We don't start counting yet, so starting from zero before calling count-router-inner-conditions makes sense to me. Also, I don't think incrementing in count-router-inner-conditions? is not omitted. The returned value was used in below step.

```
 Set |result|'s [=count router condition result/total count=] to |currentResult|'s [=count router condition result/total count=].
 ```

Reply:
I guess the sentence does not exist any more.
Let me mark this resolved.